SystemContextTest fails on Windows

Related to #1221. Seems to be a path issue. 

Exception stack trace:

```
java.lang.AssertionError: 
Expected: (an instance of java.lang.IllegalArgumentException and exception with message a string containing "Node id -1 needs to be non negative and smaller then cluster size 1.")
     but: exception with message a string containing "Node id -1 needs to be non negative and smaller then cluster size 1." message was "Illegal char <:> at index 2: /C:/camunda/repos/tngp/zeebe/broker-core/target/test-classes/io/zeebe/broker/system/\metrics/zeebe.prom"
Stacktrace was: java.nio.file.InvalidPathException: Illegal char <:> at index 2: /C:/camunda/repos/tngp/zeebe/broker-core/target/test-classes/io/zeebe/broker/system/\metrics/zeebe.prom
	at sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:182)
	at sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:153)
	at sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:77)
	at sun.nio.fs.WindowsPath.parse(WindowsPath.java:94)
	at sun.nio.fs.WindowsFileSystem.getPath(WindowsFileSystem.java:255)
	at java.nio.file.Paths.get(Paths.java:84)
	at io.zeebe.broker.system.configuration.ConfigurationUtil.toAbsolutePath(ConfigurationUtil.java:30)
	at io.zeebe.broker.system.configuration.MetricsCfg.init(MetricsCfg.java:29)
	at io.zeebe.broker.system.configuration.BrokerCfg.init(BrokerCfg.java:47)
	at io.zeebe.broker.system.configuration.BrokerCfg.init(BrokerCfg.java:40)
	at io.zeebe.broker.system.SystemContext.initSystemContext(SystemContext.java:99)
	at io.zeebe.broker.system.SystemContext.<init>(SystemContext.java:93)
	at io.zeebe.broker.system.SystemContextTest.initSystemContext(SystemContextTest.java:101)
	at io.zeebe.broker.system.SystemContextTest.shouldThrowExceptionIfNodeIdIsNegative(SystemContextTest.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:239)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:538)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:760)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:460)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:206)

	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:956)
	at org.junit.Assert.assertThat(Assert.java:923)
	at org.junit.rules.ExpectedException.handleException(ExpectedException.java:252)
	at org.junit.rules.ExpectedException.access$000(ExpectedException.java:106)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:241)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:538)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:760)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:460)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:206)
```

--- a/broker-core/src/test/java/io/zeebe/broker/system/SystemContextTest.java
+++ b/broker-core/src/test/java/io/zeebe/broker/system/SystemContextTest.java
@@ -97,7 +97,6 @@ public class SystemContextTest {
   }
 
   private SystemContext initSystemContext(BrokerCfg brokerCfg) {
-    return new SystemContext(
-        brokerCfg, SystemContextTest.class.getResource("").getPath(), new ControlledActorClock());
+    return new SystemContext(brokerCfg, "test", new ControlledActorClock());
   }
 }

--- a/qa/integration-tests/src/test/java/io/zeebe/broker/it/startup/BrokerInvalidCfgTest.java
+++ b/qa/integration-tests/src/test/java/io/zeebe/broker/it/startup/BrokerInvalidCfgTest.java
@@ -17,6 +17,8 @@ package io.zeebe.broker.it.startup;
 
 import io.zeebe.broker.Broker;
 import io.zeebe.util.sched.clock.ControlledActorClock;
+import java.net.URISyntaxException;
+import java.nio.file.Paths;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
@@ -26,10 +28,15 @@ public class BrokerInvalidCfgTest {
   @Rule public ExpectedException expectedException = ExpectedException.none();
 
   @Test
-  public void shouldThrowExceptionWithNegativePartitions() {
+  public void shouldThrowExceptionWithNegativePartitions() throws URISyntaxException {
     // given
     final String path =
-        BrokerInvalidCfgTest.class.getResource("/invalidCfgs/negativePartitions.toml").getPath();
+        Paths.get(
+                BrokerInvalidCfgTest.class
+                    .getResource("/invalidCfgs/negativePartitions.toml")
+                    .toURI())
+            .toAbsolutePath()
+            .toString();
 
     // expected
     expectedException.expect(IllegalArgumentException.class);

